packages/integration-sdk/CHANGELOG.md

```diff
@@ -40,15 +42,6 @@ import { RelationshipClass } from '@jupiterone/integration-sdk-core';
   `types` from `IntegrationStep` in favor of `entities` and `relationships`,
   which contain metadata used to generate documentation.
 
-- **BREAKING** [#283](https://github.com/JupiterOne/sdk/issues/283) - Change
```

Comment:
I originally had this breaking change in here, but I removed it from the release notes because we ultimately decided to completely remove support for `jobState.getEntity`. The removal is described in the removal section below.